app/assets/stylesheets/resources.css.scss

```diff
@@ -0,0 +1,89 @@
+.resources {
+  list-style-type: none;
+}
+
+h3 {
+  margin: 10px 0 10px 5px;
+}
+  
+.resource-cards {
+  @include span-columns(6);
+  @include omega(2n);
+  background-color: $light-tan;
+  box-shadow: 10px 10px 30px 0px rgba(0,0,0,0.75);
```

Comment:
Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`